src/shared/ui/SelectTabs/SelectTabs.jsx

```diff
@@ -0,0 +1,34 @@
+import "./SelectTabs.css";
+import cx from "classnames";
+
+const SelectTabs = ({ genres = [], selectedGenre, onSelect }) => {
+  return (
+    <div className="selector">
+      {genres.map((genre) => {
+        return (
+          <div key={genre.id}>
+            <input
+              className="selector-radio"
+              id={genre.id}
+              type="radio"
+              value={genre.value}
+              name={genre.name}
+              checked={selectedGenre === genre.value}
+              onChange={onSelect}
+            />
+            <label className="selector-item" htmlFor={genre.id}>
+              {genre.value}
+            </label>
+            <div
+              className={cx("selector-line", {
+                active: selectedGenre === genre.value,
```

Comment:
`genre.value` can be undefined, because it's possible that we get an empty array  `genres`, so it's better to check undefined possibility using the operator `?`

`genre?.value`

please, take a look https://learn.javascript.ru/optional-chaining 

<img width="591" alt="image" src="https://github.com/AnnaMartis/Epam-project/assets/36797396/d603787e-c934-49ec-804a-1327e78b6048">


<img width="689" alt="image" src="https://github.com/AnnaMartis/Epam-project/assets/36797396/02b4d6e0-5029-4cc8-961d-6b8ed37efea2">